frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx

```diff
@@ -81,9 +81,7 @@ export const sidePanelLogic = kea<sidePanelLogicType>([
             (featureFlags, isCloudOrDev, isReady, hasCompletedAllTasks) => {
                 const tabs: SidePanelTab[] = []
 
-                if (featureFlags[FEATURE_FLAGS.NOTEBOOKS]) {
-                    tabs.push(SidePanelTab.Notebooks)
-                }
+                tabs.push(SidePanelTab.Notebooks)
```

Comment:
Also Notebooks are never going to be flagged in 3000

Reply:
This actually made me realise - we need to somehow link the notebooks flag to the 3000 flag...